nci-crdc.datacommons.io/manifest.json

```diff
@@ -7,17 +7,25 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "fence": "quay.io/cdis/fence:4.12.0",
-    "arborist": "quay.io/cdis/arborist:2.3.2",
-    "google-sa-validation": "placeholder",
-    "indexd": "quay.io/cdis/indexd:2.2.0",
-    "peregrine": "quay.io/cdis/peregrine:1.0.5",
-    "pidgin": "quay.io/cdis/pidgin:1.2.0",
+    "fence": "quay.io/cdis/fence:2020.03",
+    "arborist": "quay.io/cdis/arborist:2020.03",
+    "google-sa-validation": "placeholder:2020.03",
+    "indexd": "quay.io/cdis/indexd:2020.03",
+    "peregrine": "quay.io/cdis/peregrine:2020.03",
+    "pidgin": "quay.io/cdis/pidgin:2020.03",
     "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
-    "sheepdog": "quay.io/cdis/sheepdog:1.1.6",
-    "portal": "quay.io/cdis/data-portal:2.2.5",
+    "sheepdog": "quay.io/cdis/sheepdog:2020.03",
+    "portal": "quay.io/cdis/data-portal:2020.03",
+    "sower": "quay.io/cdis/sower:2020.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "jupyterhub": "quay.io/occ_data/jupyterhub:master"
+    "metadata": "quay.io/cdis/metadata-service:2020.03",
```

Comment:
Oh I did not realize metadata service was part of the March Release, that's really cool, I'm okay with rolling it out.

Reply:
That has been copied over from DCF staging.
Checking...